src/protogui/WriteFile.java

```diff
@@ -257,33 +260,102 @@ public int generateColumn2Numbers(int problemNumbers)
         return numberColumn2;
     }
     
-    public String generateCarriedOnes(int num1, int num2)
+    public String generateCarriedNumbers(int num1, int num2)
     {
-        boolean carriedOne = false;
-        String solutionOnes = "&nbsp;&nbsp;";
+        String carriedDigits = "&nbsp;&nbsp;";
         while(num1 > 0)
+        {
```

Comment:
Could you add some additional Documentation to this. I'm not sure if Sexton is looking for details like this but it might help us understand little things.